Dart/thirdPartySrc/analysisServer/org/dartlang/analysis/server/protocol/AnalysisError.java

```diff
@@ -119,12 +119,12 @@ public boolean equals(Object obj) {
   public static AnalysisError fromJson(JsonObject jsonObject) {
     String severity = jsonObject.get("severity").getAsString();
     String type = jsonObject.get("type").getAsString();
-    String code = jsonObject.get("code") == null ? null : jsonObject.get("code").getAsString();
     Location location = Location.fromJson(jsonObject.get("location").getAsJsonObject());
     String message = jsonObject.get("message").getAsString();
     String correction = jsonObject.get("correction") == null ? null : jsonObject.get("correction").getAsString();
+    String code = jsonObject.get("code").getAsString();
```

Comment:
Per an off-line discussion, I'll manually edit these files to make the library tolerant of missing `code` fields.